src/main/kotlin/uk/gov/justice/digital/hmpps/visitscheduler/repository/VisitRepository.kt

```diff
@@ -31,4 +33,22 @@ interface VisitRepository : JpaRepository<Visit, Long>, JpaSpecificationExecutor
     startDateTime: LocalDateTime,
     endDateTime: LocalDateTime
   ): Boolean
+
+  @Query(
+    "SELECT v.visitRestriction as visitRestriction, count(v) as count  FROM Visit v " +
+      "WHERE v.prisonId = :prisonId and " +
+      "v.visitStart >= :startDateTime and " +
+      "v.visitStart < :endDateTime and " +
+      "v.visitRoom = :visitRoom and " +
+      "(v.visitRestriction = 'OPEN' or v.visitRestriction = 'CLOSED') and " +
+      " v.visitStatus in (:visitStatuses) " +
```

Comment:
I see why you have done this, if you have change code to use a visitStatuses, I would refactor my other code (line 28) above to use the same approach to be constant other wise we have two ways of doing the same thing. Consistency is very important, might not be a problem with a one off but when you get a 100 one offs over time it becomes a problem

Reply:
Fair point. I will update my code. I think having the statuses in the SQL makes it more readable and its only 2 statuses so IN does not make a lot of sense.